recipes-go/3mdeb-rtectrl/3mdeb-rtectrl_git.bbappend

```diff
@@ -1,13 +1,23 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
+check_distro_ver() {
+    local _distro_version="${DISTRO_VERSION}"
+
+    [ $(echo "$_distro_version" | grep '0.8.[0-9]*') ] && echo True || echo False
+
+    return 0
+}
+
 SRC_URI:append = " \
     file://RteCtrl.cfg \
     file://RteCtrl.service \
+    ${@ '' if bb.build.exec_func("check_distro_ver", d) else 'file://flash.sh'} \
```

Comment:
Not needed, always install it

Reply:
Dropped the commit [here](https://github.com/3mdeb/meta-rte/compare/c4c0f2b6c947061defa1b83bd585745ec0ecadb6..b09e9b726f3372dd7e3b9054bc79468968f7ece0)